### DISCUSSION TITLE
Support for blocking a sender for short duration if it crosses rate-limit set.

### DISCUSSION THREAD

**Discussion by pradeepnr** [vladimir-bukhtoyarov/bucket4j#168] [Q&A]
Hi @vladimir-bukhtoyarov,

First of all I would like to thank you for creating Bucket4j library.
We have a use-case were, if a sender crosses the rate-limit then block it for 24 hours. Can we configure that in Bucket4j?

Regards,
Pradeep

**Comment by vladimir-bukhtoyarov** [comment]
Hello @pradeepnr

 currently, this feature is not supported, but I have plans to implement it in autumn https://github.com/vladimir-bukhtoyarov/bucket4j/issues/78
